fearless/Modules/Staking/SelectValidatorsFlow/YourValidatorList/YourValidatorListViewController.swift

```diff
@@ -13,8 +36,15 @@ final class YourValidatorListViewController: UIViewController, ViewHolder {
 
     private var viewState: YourValidatorListViewState?
 
-    init(presenter: YourValidatorListPresenterProtocol, localizationManager: LocalizationManagerProtocol) {
+    let counterFormater: LocalizableResource<NumberFormatter>
```

Comment:
may we place it in presenter? you've to pass localizationManager twice: to VC and presenter

Reply:
This simplifies the display logic. We use it to display section titles. For example Elected (5) and we don't need to pass string numbers from presenter because it can be derived from the size of the list.